### DISCUSSION TITLE
Watched Replays Graph doesn't display correctly

### DISCUSSION THREAD

**Discussion by Xotna** [ppy/osu#16476] [Q&A] [answered]
Found on this profile : https://osu.ppy.sh/users/17442279

It looks like the user has 0.5 replays watched in december 2020, when hovering over this month no value is displayed.
When counting the total number of watched replays (12 according to the graph, not counting the 0.5), it doesn't match the value displayed on his profile stats (10).
Also, and this is the case for every profile I checked so far, the months displayed when hovering don't correspond to the scale (1 month discrepancy).
![image](https://user-images.githubusercontent.com/97849339/149673805-68ebf886-8842-43b1-a916-fc3447e8f74c.png)

**Comment by bdach** [comment, marked as answer]
wrong repository. you want https://github.com/ppy/osu-web

the lazer client shows the graph correctly, i've checked.

![2022-01-16-201559_2096x419_scrot](https://user-images.githubusercontent.com/20418176/149674508-20c5743d-46ac-4d9c-be11-d529b883dd89.png)

**Comment by Xotna** [comment]
Oh you're right, my bad, thanks for the answer !
